modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/robotic.dm

```diff
@@ -19,9 +19,9 @@
 	mutant_bodyparts = list()
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | ERT_SPAWN | RACE_SWAP | SLIME_EXTRACT
 	reagent_flags = PROCESS_SYNTHETIC
-	burnmod = 1.3
-	brutemod = 1.2
-	coldmod = 0.5
+	burnmod = 1.5 // Every 0.1% is 10% above the base.
+	brutemod = 1.6 
+	coldmod = 1.2
```

Comment:
:( rip

Reply:
*60%, and its fair exchange for them ignoring wounds (40% of melee combat), and three of the main damage types (blood, oxy, toxin)